src/Symfony/Bridge/Doctrine/ContainerAwareEventManager.php

```diff
@@ -136,16 +149,47 @@ public function removeEventListener($events, $listener)
         }
     }
 
+    /**
+     * Adds an EventSubscriber by its identifier in the ServiceLocator injected in the constructor.
+     */
+    public function addEventSubscriberId(string $subscriber): void
```

Comment:
oh, one question: can't we provided the $ids to the constructor instead of adding a method?

Reply:
Hmm.. we can, but the benefit of adding a method is: it's also compatible with class that extends `ContainerAwareEventManager` and override the constructor.

Reply:
Child classes can also use the constructor argument. Also, this is a big fix. Not adding any API is best.

Reply:
Let's forget about classes that extend this.

Reply:
done